scripts/solc-bump.js

```diff
@@ -13,12 +13,14 @@ const PACKAGES_DIR = path.resolve(__dirname, "..", "packages");
  **/
 function updatePackages(dryRun = false) {
   const packages = fs.readdirSync(PACKAGES_DIR);
+  const packageJsonName = "package.json";
+  const encoding = "utf8";
   return packages
     .map(packageName => {
       // parse the package.json file for `packageName`
       const packagePath = path.resolve(PACKAGES_DIR, packageName);
-      const packageJsonPath = path.resolve(packagePath, "package.json");
-      const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, "utf8"));
+      const packageJsonPath = path.resolve(packagePath, packageJsonName);
```

Comment:
I don't think there's any reason to turn `"package.json"` into a named constant instead of a plain string literal; I think it makes the code less clear, not more.  Everyone knows what `"package.json"` means; it's quite clear on its own, it's not like some magic numbers whose meaning is unclear until it's given a name.  Whereas it's a lot clear what `packageJsonName` means (and makes you think that there might be multiple possibilities for it).  Then you look up the definition and it's just `"package.json"`... it would have been clearer to do without the indirection.  And right now, we do.  I don't think we should make this change.